src/components/input-field/input-field.tsx

```diff
@@ -570,4 +574,17 @@ export class InputField {
             event.preventDefault();
         }
     }
+
+    private syncInvalidToTextField() {
+        if (!this.mdcTextField) {
+            return;
+        }
+
+        if (!this.mdcTextField.valid) {
+            return;
```

Comment:
> This sync update forces the mdcTextField into an invalid state if the invalid prop is set, otherwise it lets mdcTextField determine if the component is valid or not.

Hmm… 🤔

What happens if, for some reason, the consumer's validation says the input is valid, but the MDC component says it's not? The component would render as invalid, I assume, but the value of limel-input-field's `invalid` property would be `false, right? That might mean that a Save-button that normally becomes disabled when there are validation errors will be enabled.

Regardless, that should reasonably be a much less severe bug than the bug this fixes, but we might want to try it out so we know.

Reply:
Ya im not 100% sure how we could make the save button disabled if the MDC internal validation fails. From what i understand of this code the MDC internal validation is not exported in any way so theres no way to access it outside of limel-input-field right?